src/status_im2/contexts/activity_center/utils.cljs

```diff
@@ -2,5 +2,8 @@
 
 (defn contact-name
   [contact]
-  (or (get-in contact [:names :nickname])
-      (get-in contact [:names :three-words-name])))
+  (or
+   (get-in contact [:names :ens-name])
+   (get-in contact [:names :display-name])
+   (get-in contact [:names :nickname])
```

Comment:
Is nickname has high priority then display-name?

Reply:
Same question, I always thought the user defined `nickname` has the highest priority.

Reply:
Or in other words, if I have my mom as a mutual contact, and I set her nickname to be `Mom`, then I don't care if she has an ens name. She's `mom` after all!

Reply:
ouch! fixing  :dagger:

Reply:
done, thanks for the feedback!